src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,598 @@
+
+- A number of proxy [instances](#lms-instances) (running in separate Kubernetes pods) across which application traffic is distributed and routed to the source and target databases.
+- An "orchestrator" service (running in a single Kubernetes pod) that coordinates the proxy instances and sends the cutover commands.
+
+This page describes how to [install](#installation), [configure](#configuration), [secure](#security), and [use the LMS](#molt-lms-cli) to perform a live migration. {% comment %}For more information, see [Migration Strategy: Live Migration]({% link {{ page.version.version }}/migration-strategy-live-migration.md %}).{% endcomment %}
+
+## Terminology
+
+- A *live migration* involves maintaining two production databases (a source and target database) and replicating data between them until a final cutover.
+- The *source of truth* is the database that serves reads and writes to the application during a live migration. A cutover switches the source of truth.
+- *Shadowing* is the replication of writes from the source of truth to the target database. The LMS supports multiple [shadowing modes](#shadowing-modes).
+
+## Requirements
+
+- [Kubernetes](https://kubernetes.io/) cluster
+- [Helm](https://helm.sh/docs/intro/install/) package manager for Kubernetes
+
+#### Supported databases
+
+- [PostgreSQL]({% link {{ page.version.version }}/migrate-from-postgres.md %}) (source)
+- [MySQL]({% link {{ page.version.version }}/migrate-from-mysql.md %}) (source)
+- CockroachDB (target)
+
+## Installation
+
+The LMS definitions are located in the [`molt` repository](https://molt.cockroachdb.com/):
+
+- [`molt-lms` Helm chart](https://molt.cockroachdb.com/charts/lms/Chart.yaml)
+- [`molt-lms` values file](https://molt.cockroachdb.com/charts/lms/values.yaml) for [configuration](#configuration)
+
+When you install the `molt-lms` chart, the LMS proxy instances and orchestrator are initialized as Kubernetes pods:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+kubectl get pods
+~~~
+
+~~~
+NAME                                          READY   STATUS      RESTARTS      AGE
+lms-molt-lms-5cbd7c748-bzbgn                  1/1     Running     0             2m30s
+lms-molt-lms-5cbd7c748-g5zqf                  1/1     Running     0             2m34s
+lms-molt-lms-5cbd7c748-lbjjz                  1/1     Running     0             2m3
+lms-molt-lms-orchestrator-596d4b54d8-c4v97    1/1     Running     0             21m
+...
+~~~
+
+You will see `lms-molt-lms` pods that match the configured [number of LMS instances](#lms-instances), along with one `lms-molt-lms-orchestrator` pod.
+
+## Configuration
+
+To configure the LMS, override the values defined in [`values.yaml`](https://molt.cockroachdb.com/charts/lms/values.yaml). For information on setting Helm chart values, see the [Helm documentation](https://helm.sh/docs/helm/helm_upgrade/).
```

Comment:
As noted above, the values themselves will be in the YAML but at `molt.cockroachdb.com/lms/charts` (note that the paths have changed to be lms prefixed first since we have different binaries we are sharing inside of lms) the `.tgz` package and `index.yaml` will be.

Reply:
> the .tgz package and index.yaml will be

I'm confused on what these are. Can you elaborate?

Reply:
Yep of course, so when you do a `helm package` command, it will generate a `.tgz` zipped archive for all the YAMLs in the chart. The `index.yaml` is a manifest basically that describes the summary details of the zipped archive. Those are the items that live in the bucket  at `molt.cockroachdb.com/lms/charts`. These are just implementation details of how the chart gets published.

Reply:
The source of the charts lives on Github. That's all you need to know